caproto/threading/client.py

```diff
@@ -670,7 +672,7 @@ def command_loop(self):
                                             name, cid, *accepted_address, *new_address,
                                             extra={'pv': name,
                                                    'their_address': accepted_address,
-                                                   'our_address': self.udp_sock.getsockname()[:2]})
+                                                   'our_address': self.broadcaster.client_address})
```

Comment:
This won't work until #544 is merged, right? I don't think `client_address` exists yet.

```
$ git grep client_address
caproto/_commands.py:    .. attribute:: client_address
caproto/_commands.py:    def __init__(self, client_address='0.0.0.0'):
caproto/_commands.py:            ipv4_to_int32(str(client_address)))
caproto/_commands.py:    def client_address(self):
caproto/tests/test_serialization.py:    'client_address': [ip],
doc/source/command-line-client.rst:    [D 20:00:47.564 _broadcaster:74] 1 of 1 RepeaterRegisterRequest(client_address='0.0.0.0')
doc/source/shark.rst:   1550679067.619182 192.168.86.21:55928->255.255.255.255:5065 RepeaterRegisterRequest(client_address='0.0.0.0')
```

Reply:
I mean using `our_address` here. #544 isn't required.